CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+# 0.8.0 (T.B.D)
+
+## Migration Disclaimer
+
+Version 0.8.0 of M3 switches M3DB to use file descriptors and the Read() system call for reading data files instead of mmaps.
```

Comment:
nit: `s/reading data files/reading time series data files` to clearly distinguish between index vs ts data files.